identity/manager.go

```diff
@@ -51,6 +54,37 @@ func (idm *identityManager) GetIdentities() []Identity {
 	return ids
 }
 
+func (idm *identityManager) Unlock(identity Identity, passphrase string) error {
+	keystoreManager := idm.keystoreManager
+	accountExisting := identityToAccount(identity)
+
+	account, err := keystoreManager.Find(accountExisting)
+	if err != nil {
+		return err
+	}
+
+	err = keystoreManager.Unlock(account, passphrase)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Instead of these 5 lines of code, we could just return result of `Unlock`:
```go
return keystoreManager.Unlock(account, passphrase)
```
Unless you think it's good to keep `err != nil` check here for consistency?

Reply:
If we decide to get rid of IsLocked method then maybe we could abstract some of the errors with our own? So that packages using this manager wouldn't have to include ethereum's keystore package just for error constants.